src/utils/uploader.ts

```diff
@@ -111,12 +109,17 @@ export class BlobUploader {
         }
     }
 
-    getCommitment(blob: Uint8Array): Uint8Array {
-        return this.kzg.blobToKzgCommitment(blob);
+    async computeCommitmentsForBlobs(blobs: Uint8Array[]): Promise<Uint8Array[]> {
+        return await this.kzg.computeCommitmentBatch(blobs);
+    }
+
+    async computeVersionedHashForBlob(blob: Uint8Array): Promise<string> {
```

Comment:
This function name is confusing because its return value is actually not a versioned hash

Reply:
fix